packages/vats/src/types.js

```diff
@@ -72,13 +72,13 @@ export {};
  */
 
 /**
- * @typedef {object} ScopedBridgeManager An object which handles messages for a specific bridge
+ * @typedef {object} BridgeChannel An object which handles messages for a specific bridge
  * @property {(obj: any) => Promise<any>} toBridge
  * @property {(obj: any) => Promise<void>} fromBridge
  * @property {(handler: ERef<BridgeHandler>) => void} setHandler
  */
 
 /**
  * @typedef {object} BridgeManager The object to manage this bridge
- * @property {(bridgeId: string, handler?: ERef<BridgeHandler | undefined>) => ScopedBridgeManager} register
+ * @property {(bridgeId: string, handler?: ERef<BridgeHandler | undefined>) => BridgeChannel} register
```

Comment:
I think some names can still be improved a bit.
```suggestion
 * @property {(channelName: string, inboundHandler?: ERef<BridgeChannelInbounder | undefined>) => BridgeChannel} register Create and return a new named channel
```